tools/test_containers_compose

```diff
@@ -20,7 +20,7 @@ wait_until() {
 
 setup_containers() {
   for retry in {2..0}; do sudo docker-compose build && break; echo "Remaining retries $retry"; done
-  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d
+  sudo MOJO_CLIENT_DEBUG=1 docker-compose up -d || docker-compose logs && exit 1
```

Comment:
This will also execute the `exit 1` if the `docker-compose` command succeeds

Reply:
Here it looks you were faster (but I only saw your comment after submitting mine) :-)